scripts/spack/packages/enzyme/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
Tribol is licensed under the MIT license....is this Apache or MIT specific to spack since this is a spack package? Just clarifying.

Reply:
Updated all of these in our local spack package.py files